api/src/main/java/io/minio/MinioClient.java

```diff
@@ -98,18 +98,11 @@
 import java.security.cert.CertificateException;
 import java.security.cert.X509Certificate;
 
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Set;
-import java.util.Iterator;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.NoSuchElementException;
-import java.util.Scanner;
+import java.util.*;
```

Comment:
Its required to import specifically than all.  It is done by some code analysis tools guidelines.

Reply:
I didnt find any linter hint in the CONTRIBUTING. But sure, I can change that.

Reply:
Done (both for MinioClient and MinioClientTest)